about.html

```diff
@@ -0,0 +1,316 @@
+      <!-- Start Section Three -->
+      <section id="sectionthree about" class="sectionthree about">
+        <div id="sections__heading about" class="sections__heading about">
+          <h1
+            id="sectionthree__container__h1 about"
+            class="sectionthree__container__h1 about"
+          >
+            JustCold Events
+          </h1>
+          <span id="sections__line about" class="sections__line about"></span>
+        </div>
+        <div
+          id="sectionthree__container about"
+          class="sectionthree__container about"
+        >
+          <div
+            id="sectionthree__container__item about"
+            class="sectionthree__container__item about"
+          >
+            <div
+              id="sectionthree__container__item__p about"
+              class="sectionthree__container__item__p about"
+            >
+              <p
+                id="sectionthree__container__p about"
+                class="sectionthree__container__p about"
+              >
+                Take a look at the last two JustCold events which took place in
+                lister center mall
+              </p>
+            </div>
+            <div id="sectionthree__overlay" class="sectionthree__overlay">
+              <div
+                id="sectionthree__container__img about__1"
+                class="sectionthree__container__img about__1"
+              >
+                <div
+                  id="sectionthree__overlay__bgcolor about"
+                  class="sectionthree__overlay__bgcolor about"
+                >
+                  <h3
+                    id="sectionthree__container__h3 about"
+                    class="sectionthree__container__h3 about"
+                  >
+                    2022
+                  </h3>
+                  <p
+                    id="sectionthree__container__overlay__p about"
+                    class="sectionthree__container__overlay__p about"
+                  >
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div
+                id="sectionthree__container__img about__2"
+                class="sectionthree__container__img about__2"
+              >
+                <div
+                  id="sectionthree__overlay__bgcolor about"
+                  class="sectionthree__overlay__bgcolor about"
+                >
+                  <h3
+                    id="sectionthree__container__h3 about"
+                    class="sectionthree__container__h3 about"
+                  >
+                    2022
+                  </h3>
+                  <p
+                    id="sectionthree__container__overlay__p about"
+                    class="sectionthree__container__overlay__p about"
+                  >
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  </p>
+                </div>
```

Comment:
- [ ] The images in the mobile version should be arranged below one another according to the provided design. Please change that for a better user interface.

- Your design:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/200509777-de5c10cd-9214-4d2a-95a1-79a9ac6663c8.png">

- Original design:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/77190203/200509948-cf7d7dd6-e8c1-47c9-b87f-0236c1415392.png">